utils/rowgroup/rowaggregation.cpp

```diff
@@ -684,6 +684,12 @@ void RowAggregation::initialize()
 
     config::Config* config = config::Config::makeConfig();
     string tmpDir = config->getTempFileDir(config::Config::TempDirPurpose::Aggregates);
+    bool compress = false;
+    string compStr = config->getConfig("RowAggregation", "Compression");
+    if (boost::iequals(compStr, "snappy"))
```

Comment:
iequals internally takes a global kernel lock getting locale.

Reply:
Right, forgot about that. Fixed.